ai/recommender/knn_meta_recommender.py

```diff
@@ -141,7 +142,7 @@ def recommend(self, dataset_id, n_recs=1, dataset_mf = None):
                            if subset[i]]) 
                 logger.info(f'btw, there are {num_results} results for {dataset_id} already')
             ml_rec, p_rec, rec_score = (ml_rec[:n_recs],
-                    [self.param_htable[int(p)] for p in phash_rec[:n_recs]],
+                    [self.hash_2_param[p] for p in phash_rec[:n_recs]],
```

Comment:
Please use `git cherry-pick a6957f81f5715eeac2262820b86a6bc1f6d8bbd7` to fix a bug in KNNMetaRecommender. (see this commit a6957f81f5715eeac2262820b86a6bc1f6d8bbd7)